docs/guides/argo-cd-crossplane.md

```diff
@@ -7,32 +7,54 @@ weight: 270
 are a great combination. Argo CD provides GitOps while Crossplane turns any Kubernetes
 cluster into a Universal Control Plane for all of your resources. There are
 configuration details required in order for the two to work together properly.
-This doc will help you understand these requirements. It is recommended to use
+This doc will help you understand these requirements.
 
-Argo CD version 2.4.8 or later with Crossplane.
- 
-Argo CD synchronizes Kubernetes resource manifests stored in a Git repository
-with those running in a Kubernetes cluster (GitOps). There are different ways to configure 
+It is recommended to use Argo CD version 2.4.8 or later.
 
-how Argo CD tracks resources. With Crossplane, you need to configure Argo CD 
-to use Annotation based resource tracking. See the [Argo CD docs](https://argo-cd.readthedocs.io/en/latest/user-guide/resource_tracking/) for additional detail.
- 
-### Configuring Argo CD with Crossplane
- 
-To configure Argo CD for Annotation resource tracking, edit the `argocd-cm`
-
-`ConfigMap` in the `argocd` `Namespace`. Add `application.resourceTrackingMethod: annotation`
-
-to the data section as below:
+### Resource Tracking with annotations
 
+In order for Argo CD to properly track Crossplane generated resources, tracking method must be updated to annotations :

```

Comment:
Could you please wrap docs at 80 chars, in line with the existing file. There's a few plugins that will do this.

Reply:
@negz have we considered adding a linter to the CI pipeline to check and report on this? If not I can open an issue to look to introduce this.

Reply:
@jeanduplessis Happy to have a linter handle this, please go ahead.